setup.py

```diff
@@ -343,7 +343,7 @@ def run(self):
         "recommendation",
     ],
     ext_modules=extensions,
-    install_requires=["numpy", "scipy", "tqdm", "powerlaw"],
+    install_requires=["numpy", "scipy", "tqdm", "powerlaw","bottleneck"],
```

Comment:
Do we need `bottleneck` for all models?
If not, please remove `bottleneck` from this argument.

Reply:
I've removed bottleneck